### DISCUSSION TITLE
Allow to filter for more than one country / page / source etc

### DISCUSSION THREAD

**Discussion by mforchner** [plausible/analytics#1528] [Anything Else]
Dear team at Plausible,

we are using the latest version of the self-hosted Plausible variant. Our marketing team regularly needs to aggregate data for a group of countries, e.g. Germany, Switzerland and Austria (DACH), South America etcetera. Currently it seems to be possible to filter for one country only:

<img width="447" alt="Bildschirmfoto 2021-12-10 um 10 22 52" src="https://user-images.githubusercontent.com/12170858/145549998-91cafcba-27c9-4055-b628-64b70680411f.png">

Would it be possible to allow for multiple country filters? I could not find a discussion on that topic both in this forum and in the issues backlog.

All the best from Berlin!
Marcus

**Update**: You can now filter your stats by multiple countries, pages, sources and more, all at once. Click on the "Filter" button in your dashboard to start exploring.

**Comment by RobertJoonas** [comment]
While it's not yet possible in the dashboard, this feature is already supported in Stats API. See: https://plausible.io/docs/stats-api#filtering

On another note, it would be nice to also allow:

1) excluding multiple sources/countries/etc with the `is_not` filter type in the dashboard
2) exlcuding multiple sources/countries/etc in the Stats API (e.g. `visit:country!=US|DE|GB`)

**Comment by JohnSharkey** [comment]
As it is needed for SEO reporting for search traffic, it is also needed to filter all search engines to see data, perhaps by a source "type" somehow or by selecting known search engines.

**Reply by sjoerd-st** [reply]
Agreed. And needed for the exact same purpose, reporting on "Organic" traffic, which I assume is a rather common use case.
Unless I'm missing something in the UI, there doesn't seem to be a way to create an "or" type of filter, where I can group:
Source =
- Google
- Bing
- DuckDuckGo

etc.
It would greatly improve my experience.

**Comment by metmarkosaric** [comment]
**Update**: You can now filter your stats by multiple countries, pages, sources and more, all at once. Click on the "Filter" button in your dashboard to start exploring.

**Comment by JohnSharkey** [comment]
Great news!! Well done!! :)

On Mon, Mar 27, 2023 at 3:30 PM Marko Saric ***@***.***>
wrote:

> *Update*: You can now filter your stats by multiple countries, pages,
> sources and more, all at once. Click on the "Filter" button in your
> dashboard to start exploring.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/plausible/analytics/discussions/1528#discussioncomment-5442092>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZD3SCFQ5U5IG25IWYEA2GLW6GQADANCNFSM5JYUZU3Q>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
John Sharkey
Clear Moon Studio <https://www.clearmoonstudio.com/>
01252 703605
07715 985840
<https://clearmoonstudio.com/>

**Reply by metmarkosaric** [reply]
glad you like it @JohnSharkey, thank you!
